src/js/Components/LottoWinForm.js

```diff
@@ -0,0 +1,74 @@
+import Component from "../Core/Component.js";
+import { SET_WINNING_NUMBERS, SET_BONUS_NUMBER } from "../modules/actions.js";
+import { setWinningNumbers, setBonusNumber } from "../modules/creator.js";
+import { store } from "../modules/store.js";
+import { $ } from "../utils/dom.js";
+
+const actionMap = {
+  SET_WINNING_NUMBERS: (numArr) => {
+    store.dispatch(setWinningNumbers(numArr));
+  },
+
+  SET_BONUS_NUMBER: (num) => {
+    store.dispatch(setBonusNumber(num));
+  },
+};
+
+export default class LottoWinForm extends Component {
+  constructor(target) {
+    super(target);
+  }
+
+  setEvent(target) {
+    this.buttonEvent(target);
+  }
+
+  buttonEvent() {
+    $("#show-result-btn").addEventListener("click", () => {
+      const numArr = [];
```

Comment:
```suggestion
      const numArr = new Set()
```
중복 방지를 고려해서 Set 객체를 활용하면 좋을꺼 같습니다.